docs/pages/server-access/introduction.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Server Access Guide
+description: Introduction to features and benefits of Teleport Server Access
+---
+
+# Server Access
+
+Teleport Server Access consolidates SSH access across all environments, decreases configuration complexity, supports industry best practices and compliance while giving complete visibility over all sessions and events.
+
+Teleport Server Access is designed for the following kinds of scenarios:
+
+1. When up to a vast number of clusters must be managed using the command-line (`tsh`) or programmatically (through the Teleport API) and you want to simplify your stack, security, and configuration complexity.
+2. When it's important to track and audit every user session. 
+3. When Teleport users require a complete, dedicated, and secure SSH option (Teleport Node running in SSH mode) and more than a certificate authority (Teleport Auth) with proxy (Teleport Proxy).
+4. When resource and network security must be maximized: SSH certificates over secret keys, Two-Factor Authentication (2FA), Single Sign-On (SSO), and short-lived certificates.
+
+## Features
+
+Server Access features are organized into three main offerings: 
+
+1. **Simplified/unified configuration and stack**: 
+  - Multi-cluster management through a clean user interface and API when managing your many resources or devices.
+  - Teleport Server Access allows full SSH management without third-party dependencies. 
+  - Add functionality to Teleport through plugins.
+  - [Reduce the likelihood of a dependency or tool vulnerability](https://www.cse.usf.edu/~xou/publications/esorics08.pdf) by reducing the complexity of your stack.
+2. **Security and compliance**:
+  - 2FA and One-Time Pass (OTP) support.
+  - Role-Based Access Control (RBAC): built-in support for delegating permissions and access by the *Principle of Least Privilege* (POLP).
+  - Support for OAuth 2.0, Open ID Connect, SAML, and SSO out-of-the-box.
+  - Access Requests that can be approved or denied to define a compliance workflow.
+3. **Complete visibility into access and behavior**:
+  - Single and multi-user session auditing and recording.
+  - Configure event streaming to record and send events to specific team members or destinations.
+  - Unified resource management and [cluster introspection](https://goteleport.com/blog/openssh-vs-teleport/).
+  - A convenient web UI for easily representing and interacting with resources and user sessions.
+
+## Demo
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/addnode.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/addnode.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+## Getting started
+
+Get started using Teleport Server Access in [10 minutes](./getting-started.mdx).
+
+## Guides
+
+- [SSH](./guides/teleport-ssh.mdx)
```

Comment:
Let's see if we can use the same layout `tocless-doc` as linked above and use tiles as in our root intro with most common use-cases here:

https://goteleport.com/docs/